lib/ansible/modules/cloud/cloudstack/cs_instance_facts.py

```diff
@@ -265,9 +268,20 @@ def get_result(self, instance):
                     affinity_groups.append(affinitygroup['name'])
                 self.result['affinity_groups'] = affinity_groups
             if 'nic' in instance:
+                network_ip_addresses = defaultdict(list)
                 for nic in instance['nic']:
+                    networkname = nic['networkname']
                     if nic['isdefault'] and 'ipaddress' in nic:
                         self.result['default_ip'] = nic['ipaddress']
+                    for nic_section in itertools.chain((nic, ), nic['secondaryip']):
```

Comment:
Could you please amend this commit adding `Co-authored-by: Sviatoslav Sydorenko <wk@sydorenko.org.ua>` trailed to long commit description?